actualizacion navbars (Responsive)

haywill20 opened the pull request:
agregué Navbar2 Navbar3(Navbar 1 lo quite, me dio un problema)

haywill20 committed:
Update index.js

haywill20 committed:
Merge branch 'master' into develop

haywill20 committed:
primer commit

haywill20 committed:
actualizaciones de AboveNavBar

se elimino el arhivo Navbar1 y se volvio a utilizar el archivo AboveNavBar

haywill20 committed:
actualizaciones

haywill20 committed:
actualizacion eliminacion de titulos

haywill20 committed:
actualizacion navbar2

haywill20 committed:
actualizacion

haywill20 committed:
actualizacion home

dave-flr reviewed (changes requested):
@haywill20 Cuando hagas un cambio no hagas una copia del archivo.

- [ ] Los nuevos componentes deben seguir el patrón de los anteriores, si empiezan con mayúsculas, igualmente deben empezar con mayúsculas
- [ ]  No crees una copia del mismo componente para trabajar en el, modifica el que ya tenés sin miedo, después de todo es tu propia copia.

Actualmente hay 3 errores que tenés que reparar antes de hacer pull otra vez

- Unifica los componentes que están repetidos "Navbar2" y "navbar2" son lo mismo, y asi con los siguientes Navbars

- Como los componentes están copiados el archivo que está buscando es index.scss, pero solo está el archivo index.css en la linea 4.

- Asegurarte que no estés repitiendo código ya que al final eso es lo que deberías tratar de evitar

dave-flr commented:
Otra cosa muy importante @haywill20 procurá que los nombres de tus commits sean mas descriptivos de lo que se hizo en ese commit, no solo pongas "actualizaciones" ó "se hicieron actualizacion de ---", asi como tambien los nombres de tus pull request, explica que cambiaste, porque lo cambiaste de manera clara y precisa  :+1:

haywill20 committed:
reutilizacion de archivos(NavBar, SubNavbar)

se eliminaron los componentes, NavBar2, NavBar3,NavBar4 el codigo de estos fue agregado a los componentes que habia creado antes agustin(Navbar, SubNavBar)

jonariv09 reviewed (commented):
Esta mal este cambio, utiliza tus componentes, y sobre ellos hace tus propios cambios, los mios los deje de lado porque de ese trabajo te estas encargando vos.
A partir de ahora no toques los componetes Navbar que yo habia creado, para evitarme bastante confusiones de merge.

jonariv09 commented:
- La creacion de slider1, slider2, slider3 fue completamente inncesaria, no repitas componentes, ya tenias Slider1, Slider2, Slider3.
- Cambiarle el nombre a los componentes Slider1, Slider2, Slider3, informacion, no son nombres para nada descriptivos y no cumplen con la convencion PascalCase que he venido usando todo este tiempo para los componentes
- No uses .css en las extensiones de archivos de estilos, usa .scss.
- Crea nombres descriptivos para las clases css de los componentes, (texto, ancho, dim, text-tam, dimen) los usas sin orden.
- Mantene al dia tu fork, siempre que vayas a trabajar consulta a los demas desarrolladores acerca del estado de los cambios.
Tenes un componente seccion3 que yo le cambie a Announcement, y sigue ahi en tu fork, eso quiere decir que no hiciste pull a los cambios o no estoy tan seguro de que hiciste.

jonariv09 commented:
Este PR tiene muchos errores voy a cerrarlo, realiza todos los cambios y sugerencias que se te hicieron y envialo en un PR nuevo.
